lib/active_model_serializers/railtie.rb

```diff
@@ -30,8 +29,8 @@ class Railtie < Rails::Railtie
     # inheritable, we don't want to set it on `ActiveModel::Serializer` directly unless
     # we want *every* serializer to be considered cacheable, regardless of specifying
     # `cache # some options` in a serializer or not.
-    initializer 'active_model_serializers.caching' => :after_initialize do
-      ActiveModelSerializers.config.cache_store     = ActionController::Base.cache_store
+    initializer 'active_model_serializers.caching', :after => 'action_controller.set_configs' do
+      ActiveModelSerializers.config.cache_store     = Rails.configuration.action_controller.cache_store
```

Comment:
I think this is still useful. I think it is probably rare that someone is configuring the `config.action_controller.logger` differently that the Rails.logger. But it is somewhat more likely that they would have a different cache store for controllers than the base Rails.cache. And it does make sense that AMS uses whatever ActionController is using if that is different than Rails.cache.
One thing to note here is that the `ActiveModelSerializers.config.cache_store` is *always* being updated to match ActionController. It is not doing `||=`. I don't know if you would want to support allowing someone to specifically configure a different cache store for AMS.